Sources/AsyncHTTPClient/FoundationExtensions.swift

```diff
@@ -64,3 +64,14 @@ extension HTTPClient.Body {
         return self.bytes(data)
     }
 }
+
+@available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
+extension HTTPClientResponse {
+    /// Response body as `Data`.
+    public var data: Data? {
+        get async throws {
+            var bytes = try await bytes
```

Comment:
Mild nit: our style is to use `self.` wherever relevant.

Reply:
Done